service/src/main/resources/application.yml

```diff
@@ -32,9 +32,25 @@ spring:
     password: ${springMailPassword:"none"}
     port: 465
     protocol: 'smtps'
-    dev-mode: true
+    devMode: true
     debug: true
-    test-connection: false
+    testConnection: false
+  template:
```

Comment:
`template`, `templateNames`, `localization` are not connected with spring mail config so they shouldn't be located under `spring.mail.`

I suggest to extract them to `email.template` and `email.location` properties.